_posts/en/newsletters/2025-03-21-newsletter.md

```diff
@@ -0,0 +1,227 @@
+  in the [Using Podman Containers for Regtest Bitcoin Development][podman
+  bitcoin blog] blog post.
+
+- **Explora transaction visualization tool:**
+  [Explora][explora github] is a web-based explorer for visualizing and
+  navigating between transaction inputs and outputs.
+
+- **Hashpool v0.1 tagged:**
+  [Hashpool][hashpool github] is a [mining pool][topic pooled mining] based on
+  the [Stratum v2 reference implementation][news247 sri] where mining shares are
+  represented as [ecash][topic ecash] tokens (see [Podcast #337][pod337 hashpool]).
+
+- **DMND launching pooled mining:**
+  [DMND][dmnd website] is launching Stratum v2 pooled mining, building on their
+  previous solo mining [announcement][news281 demand].
+
+- **Krux adds taproot and miniscript:**
+  [Krux][news273 krux] adds [miniscript][topic miniscript] and taproot support,
+  leveraging the [embit][embit website] library.
+
+- **Source-available secure element announced:**
+  The [TROPIC01][tropic01 website] is a secure element built on RISC-V with
+  an [open architecture][tropicsquare github] for auditability.
+
+- **Nunchuk launches Group Wallet:**
+  [Group Wallet][nunchuk blog] supports [multisignature][topic
+  multisignature] signing, taproot, coin control, [Musig2][topic musig], and
+  secure communication between participants by repurposing the output descriptors in the
+  [BIP129][] Bitcoin Secure Multisig Setup (BSMS) file.
+
+- **FROSTR protocol announced:**
+  [FROSTR][frostr github] uses the FROST [threshold signature scheme][topic
+  threshold signature] to achieve k-of-n signing and key management for nostr.
+
+- **Bark launches on signet:**
+  The [Bark][new325 bark] implementation of [Ark][topic ark] is now
+  [available][second blog] on [signet][topic signet] with a faucet and demo
+  store for testing.
+
+- **Cove Bitcoin wallet announced:**
+  [Cove Wallet][cove wallet github] is an open source Bitcoin mobile wallet based on
+  BDK that supports technologies like PSBTs, [wallet labels][topic wallet labels],
+  hardware signing devices, and more.
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+FIXME:harding to update or remove Thursday
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #31649][] removes all checkpoint logic, replacing it with the
```

Comment:
"replacing it with" in this context sounds like this PR is replacing it, but really presync was a few years ago. Id suggest some slight rewording here @Gustavojfe